interface/i18n/mist.nl.i18n.json

```diff
@@ -171,12 +205,15 @@
                 "enterPassword": "Voer wachtwoord in om de transactie te bevestigen",
                 "unlocking": "Bevestigen...",
                 "createContract": "Contract aanmaken",
-                "estimatedFee": "Ingeschatte kosten",
-                "estimatedGasError": "Data kon niet worden uitgevoerd, dus het zal alle mogelijke gas gebruiken.",
+                "estimatedFee": "Ingeschatte transactie kosten",
+                "estimatedGasError": "Het lijkt erop dat deze transactie zal mislukken. Als het het toch verzend zou het alle verzonden gas kunnen verbruiken.",
```

Comment:
Something like `De kans is groot dat de transactie zal mislukken.`.
The last sentence isn't correct Dutch.